src/apps/viewer/app.ts

```diff
@@ -423,6 +423,28 @@ export class Viewer {
         });
     }
 
+    async loadFullResolutionMap(emdbId: string, iso: number) {
```

Comment:
```suggestion
    async loadFullResolutionEMDBMap(emdbId: string, options: { isoValue: Volume.IsoValue }) {
```

This will allow to add more options in the future more easily if needed.